Assets/MRTK/Tools/MSBuild/Scripts/CSProjectInfo.cs

```diff
@@ -243,14 +243,16 @@ private void ProcessSourceFile(SourceFileInfo sourceFile, string sourceIncludeTe
             switch (sourceFile.AssetLocation)
             {
                 case AssetLocation.BuiltInPackage:
-                    relativeSourcePath = sourceFile.File.FullName;
```

Comment:
Intentional that we're now ignoring any path for `AssetLocation.BuiltInPackage`?

Reply:
My understanding is that assignment actually does not do anything useful - if you look above `relativeSourcePath` seems to be a local variable and given we return immediately the assignment seems unnecessary.

Reply:
Ah true! Good call